README.md

```diff
@@ -1,2280 +1,31 @@
 # Open MPI
```

Comment:
I think we want something in README.md, if only so it shows up on GitHub and the like?  Seems weird to not remove HACKING.md, but to remove README.md.

Reply:
Hmm.  I left a stub README.md -- but it must have gotten lost in the rebases/conflict fixes/etc.  I'll restore.